tests/k8s.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+USTERLINK#!/usr/bin/env bash
```

Comment:
This seems like an accidental cut-n-paste?
```suggestion
#!/usr/bin/env bash
```